src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -142,8 +162,7 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
      */
     saveContent(): void {
         this.dotGlobalMessageService.loading(this.dotMessageService.get('dot.common.message.saving'));
-        this.dotEditPageService
-            .save(this.pageState.page.identifier, this.dotEditContentHtmlService.getContentModel())
+        this.pageServiceSave()
             .takeUntil(this.destroy$)
```

Comment:
We can move the takeUntil to the method and also... should be better a ```take(1)``` instead?